apache/superset
# fix: Issue #12004 timegrain not visibile

### SUMMARY
Fixes an issue that makes the SelectControl options hidden when overflowing.

### TEST PLAN
1. Go to a Dashboard
2. Inside a Filter box open the options of any single-choice dropdown (multi-options dropdowns already worked properly)
3. Make sure the options are visible even when overflowing the filter box container 

### ADDITIONAL INFORMATION


- [x] Has associated issue: #12004
